tests/conftest.py

```diff
@@ -165,11 +165,12 @@ def __default_task():
         properties={"default_data": "In memory Data Source"},
     )
     return Task(
-        "foo",
-        print,
-        [input_ds],
-        [output_ds],
-        None,
+        config_id="foo",
```

Comment:
Indeed, it is my bad. 
I moved in taipy-core the parameter `properties` from the last position to the second one. 

Thx joao!